build-logic/src/main/kotlin/ktlint-publication.gradle.kts

```diff
@@ -97,6 +97,7 @@ signing {
     val signingPassword = System.getenv("ORG_GRADLE_PROJECT_signingKeyPassword")
     useInMemoryPgpKeys(signingKeyId, signingKey, signingPassword)
 
+    val disableSigning = project.findProperty("ktlint.disable-signing") == "true"
```

Comment:
Add comment and explain why and for who this property is added. This line is very vulbnerable for removal in the future as it is not used internally in ktlint. Also I would like change the name and behavior of the property to `ktlint.publication.signing.enable`. When the property is absent or set to another value then "false" it should be considered as enabled.

Reply:
done and tested